service/rpc2/client.go

```diff
@@ -354,6 +360,20 @@ func (c *RPCClient) SetReturnValuesLoadConfig(cfg *api.LoadConfig) {
 	c.retValLoadCfg = cfg
 }
 
+func (c *RPCClient) IsMulticlient() bool {
+	var out IsMulticlientOut
+	c.call("IsMulticlient", IsMulticlientIn{}, &out)
+	return out.IsMulticlient
+}
+
+func (c *RPCClient) Disconnect(cont bool) error {
+	if cont {
+		out := new(CommandOut)
+		c.client.Go("RPCServer.Command", &api.DebuggerCommand{Name: api.Continue, ReturnInfoLoadConfig: c.retValLoadCfg}, &out, nil)
```

Comment:
I know at least for Linux if we `PTRACE_DETACH` it will continue the process automatically. Continuing first before detach could cause a later `PTRACE_DETACH` to fail. I don't think we have this guarantee on other platforms though.

Reply:
Got it.